sdk/resourcemanager/Azure.ResourceManager/README.md

```diff
@@ -53,6 +53,25 @@ using Azure.ResourceManager.Resources;
 ArmClient client = new ArmClient(new DefaultAzureCredential());
 ```
 
+Note: if you want to authenticate with the azure in China, you can use the following code:
+```
```

Comment:
This must be a snippet in order to ensure that it can be compiled and does not rot.

Reply:
Fixed